### DISCUSSION TITLE
strongSwan 6 release date?

### DISCUSSION THREAD

**Discussion by feca67** [strongswan/strongswan#1850] [Q&A] [answered]
We have completed an in-house evaluation of strongSwan 6.0 Beta3 and are pleased with the results.

Is there a proposed date for the release of V6.0.0 ? Failing that any info on timescales would be appreciated.

Apologies if this info is available online, I've looked but not found anything....

Cheers

**Comment by strongX509** [comment, marked as answer]
I'm going to release strongSwan 6.0 Beta4 in the next couple of weeks. It will be based on the PQC algorithms supported by the liboqs 0.8 library. Unfortunately NIST hasn't released any draft standards for the 2022 selected algorithms yet. Especially no OIDs for the signature algorithms have been defined yet and all KEM algorithms would have to be assigned code points form the private range of the IKEv2 Key Exchange transforms.

Unrelated to the PQC capability we would like to port a couple of features from the old deprecated **`stroke`** interface to the **`vici`** interface, so that in the stable strongSwan 6.0 release we could further decrease the support of **`stroke`** and the **`ipsec`** tools. I'm still optimistic that strongSwan 6.0 will be released before the end of this year.

**Reply by m5aquib** [reply]
`I'm still optimistic that strongSwan 6.0 will be released before the end of this year.`
Since new version could not be released last year. Any update on the new timeline?

**Comment by feca67** [comment]
Thank you for the information - very helpful, much appreciated
